packages/app-page-builder-theme/src/components/Navigation.tsx

```diff
@@ -23,14 +15,14 @@ const Navigation = ({ data }) => {
                             {item.title}
                             <ul>
                                 {item.children.map((item, index) => {
-                                    return <li key={item.id + index}>{getLink(item)}</li>;
+                                    return <Link to={item.url} key={item.id + index}><li>{item.title}</li></Link>;
```

Comment:
I think this will produce invalid HTML, since you'd get `<ul><a><li>...</li></a></ul>`. Only `li` is allowed inside of `ul`, so we need a different approach here. Maybe create an class using emotion to set `li` padding to 0 (or something along those lines)?